src/PresentationalComponents/Cards/OverviewDashbarCard/components/DashbarCardTitle/TitleWithTooltip.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import { Title } from '@patternfly/react-core';
+import { QuestionTooltip } from '../../../../Common/Common';
+
+// this component returns the title with an appropriate formatted tooltip message that is displayed when hovering over the question mark icon found next to it
+export const TitleWithTooltip = ({ title, tooltipMessage }) => {
+  return (
+    <Title headingLevel="h6" size="md">
+      {title} {QuestionTooltip(tooltipMessage)}
```

Comment:
We should correct the `QuestionTooltip` function to make it a proper component and make sure it is called as such, like here it should be:

```js
{title} <QuestionTooltip text={tooltipMessage} />
```

Reply:
Fixed 👍🏼